drgn/internal/cli.py

```diff
@@ -176,6 +176,8 @@ def main() -> None:
     init_globals: Dict[str, Any] = {"prog": prog}
     if args.script:
         sys.argv = args.script
+        if os.path.isfile(args.script[0]):
+            sys.path.insert(0, os.path.dirname(args.script[0]) or ".")
```

Comment:
It looks like the `python` command uses the absolute path of the parent directory:

```console
[osandov@relinquished dir]$ cat ../test.py
import sys
print(sys.path)
[osandov@relinquished dir]$ python3 ../test.py
['/home/osandov', '/usr/lib/python310.zip', '/usr/lib/python3.10', '/usr/lib/python3.10/lib-dynload', '/usr/lib/python3.10/site-packages']
```

If the script changes directory, this would result in different behavior. So I think we want:

```suggestion
            sys.path.insert(0, os.path.dirname(os.path.abspath(args.script[0])))
```